breakerbox-turbine/src/main/java/com/yammer/breakerbox/turbine/model/App.java

```diff
@@ -0,0 +1,30 @@
+
+package com.yammer.breakerbox.turbine.model;
+
```

Comment:
These models are pretty specific to marathon. Can we move these under a marathon specific package within `breakerbox/turbine/model/marathon` perhaps?